### ISSUE TITLE
Makefile windws 10

### ISSUE THREAD

**Issue by brunotrevisan** [libretro/pcsx_rearmed#399]
Hi Good night everyone, I'm trying to compile this version on windows 10 64b using git bash, but there are hundreds of errors in the makefile several files like LibFlac's error, what would be the correct way to compile on windows for windows?

command: $ make -f Makefile.libretro platforma=win CC=gcc

error:
gcc  -DGIT_VERSION=\"" 3c4ac5b"\" -D__WIN32__ -DNO_DYLIB  -D_FILE_OFFSET_BITS=64 -Wall -Iinclude -ffast-math -O2 -DNDEBUG -Ideps/zlib -Ideps/lightning/include -Ideps/lightrec -DLIGHTREC -DLIGHTREC_STATIC -Ideps/mman -DGPU_PEOPS -Ideps/libchdr -DHAVE_FSEEKO -Ideps/crypto -Ideps/flac-1.3.2/include -Ideps/flac-1.3.2/src/libFLAC/include -Ideps/flac-1.3.2/src/libFLAC/include -Ideps/lzma-16.04/C -DHAVE_CHD -D'PACKAGE_VERSION="1.3.2"' -DFLAC__HAS_OGG=0 -DFLAC__NO_DLL -DHAVE_LROUND -DHAVE_STDINT_H -DHAVE_STDLIB_H -DFLAC__NO_DLL -D_7ZIP_ST -Ilibretro-common/include -DFRONTEND_SUPPORTS_RGB565 -DHAVE_LIBRETRO -DNO_FRONTEND -c -o deps/lightrec/recompiler.o deps/lightrec/recompiler.c
deps/lightrec/recompiler.c:24:21: fatal error: pthread.h: No such file or directory
 #include <pthread.h>
                     ^
compilation terminated.
make: *** [deps/lightrec/recompiler.o] Error 1

**Comment by brunotrevisan** [comment]
![image](https://user-images.githubusercontent.com/27830830/78819103-0d23b080-79ac-11ea-800a-41abe28e9dc8.png)
I managed to compile it, but it was missing the background images when loading the games, the Bios boot doesn’t show the PSX logo, see this image the clock tower doesn’t appear

**Comment by dankcushions** [comment]
BIOS logo doesn't appear by default. you can enable it in the core options, but it can cause issues.

i don't know about that game, but you could try running it in interpreter mode. i'm not sure what it defaults to, but the various dynarecs can have issues. again, in core options.

**Comment by brunotrevisan** [comment]
yes i know that in the coreoptions it is disabled, but with it enabled the normal boot appears and on the black screen the PSX LOGO does not appear it is only the texts
